.github/workflows/stale.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: 'Close stale issues and PRs'
+on:  # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it
+            has not had recent activity. It will be closed if no further
+            activity occurs during the next 14 days or if the stale label
+            is not removed.
+          stale-pr-message: >
+            This PR has been automatically marked as stale because it has
+            not had recent activity. It will be closed if no further
+            activity occurs during the next 14 days or if the stale label
+            is not removed.
+          close-issue-message: >
+            This issue has been automatically closed because it has not
+            had recent activity.
+          close-pr-message: >
+            This PR has been automatically closed because it has not had
+            recent activity.
+          days-before-stale: 90
+          days-before-close: 14
+          exempt-issue-labels:
+            - bug
```

Comment:
I'm getting the following message: `A sequence was not expected`

I think it's expecting a string, see the [doc](https://github.com/actions/stale#exempt-issue-labels). There is also an example available in the [usage](https://github.com/actions/stale#usage) section.

Reply:
Thanks, I pushed a fix.